pkg/plugins/trigger/pull-request_test.go

```diff
@@ -25,17 +25,78 @@ import (
 	"github.com/jenkins-x/lighthouse/pkg/labels"
 	"github.com/jenkins-x/lighthouse/pkg/launcher/fake"
 	"github.com/jenkins-x/lighthouse/pkg/plugins"
+	"github.com/jenkins-x/lighthouse/pkg/repoowners"
 	fake2 "github.com/jenkins-x/lighthouse/pkg/scmprovider/fake"
 	"github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/util/sets"
 )
 
+type fakeOwnersClient struct{}
```

Comment:
ToDo: This should be in some test helper package as it's now being reused across tests in separate packages.